google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BucketIamSnippets.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.storage.snippets;
+
+import com.google.cloud.Identity;
+import com.google.cloud.Policy;
+import com.google.cloud.Role;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * This class contains Bucket-level IAM snippets for the {@link Storage} interface.
+ */
+public class BucketIamSnippets {
+
+  public BucketIamSnippets() {
+  }
+
+  /**
+   * Example of listing the Bucket-Level IAM Roles and Members
+   */
+  public Policy listBucketIamMembers(String bucketName) {
+    // [START view_bucket_iam_members]
+    // Initialize a Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // Get IAM Policy for a bucket
+    Policy policy = storage.getIamPolicy(bucketName);
+
+    // Print Roles and its identities
+    Map<Role, Set<Identity>> policyBindings = policy.getBindings();
+    for(Map.Entry<Role, Set<Identity>> entry : policyBindings.entrySet()) {
+        System.out.printf("Role: %s Identities: %s\n", entry.getKey(), entry.getValue());
```

Comment:
@shinfan, this is a sample. I feel the print statement makes sense. I'm open to interpretation.